package.json

```diff
@@ -15,12 +15,10 @@
     "babel-preset-airbnb": "^1.0.1",
     "babelify": "^7.2.0",
```

Comment:
If we're taking this opportunity to cleanup deps:
babelify
browserify
watchify
karma-browserify 

are probably not being used also